WeartGloves/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+#/*
+# Copyright 2021 Istituto Italiano di Tecnologia (IIT)
+# @author Ehsan Ranjbari <ehsan.ranjbari@iit.it>
+#* CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+#*/
+cmake_minimum_required(VERSION 3.5)
+
+project(YARPWeArtGlove VERSION "0.0.1"
+                  LANGUAGES CXX C)
+
+set(WeArtGloveLibName "WeArtGloveLib")
+set(WeArtGloveDevName "WeArtGlove")
+
+find_package(YARP REQUIRED)                
+find_package(${WeArtGloveLibName} REQUIRED)
+find_package(IWear REQUIRED)
+find_package(Eigen3 REQUIRED)
+find_package(iDynTree REQUIRED)
+find_package(BipedalLocomotionFramework 0.9.0
+  COMPONENTS VectorsCollection IK ParametersHandlerYarpImplementation
+             ContinuousDynamicalSystem ManifConversions
+             ParametersHandlerYarpImplementation REQUIRED)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
This variable should not be hardcoded, if you want to use C++14 you can specify it for a specific target via `target_compile_features(WeArtGlove PRIVATE cxx_std_14)`.